source/geppetto/4.2/index.md

```diff
@@ -0,0 +1,286 @@
+---
+layout: default
+title: "Geppetto"
+subtitle: "Introduction to Geppetto"
+canonical: "/geppetto/latest/geppetto.html"
+---
+[geppetto_install]: ./images/geppetto_install.png
+[IDE_overview]: ./images/IDE_overview.png
+[prev]: ./images/geppettoprev.png
+[add]: ./images/geppettoaddpuppetdb.png
+[events]: ./images/geppettoevents.png
+[refresh]: ./images/geppettorefresh.png
+[git_toolbar]: ./images/git_toolbar.png
+[os_metadatajson]: /puppet/latest/reference/modules_publishing.html#operating-system-compatibility-in-metadatajson
+[module_names]: /puppet/latest/reference/modules_publishing.html#a-note-on-module-names
+[dependencies_metadata]: /puppet/latest/reference/modules_publishing.html#dependencies-in-metadatajson
```

Comment:
The final three references (starting with `os_metadatajson`) are never used in the text. Either the links are missing and should be added, or the references should be removed.